apps/web-connect/src/features/hooks/web-buy-keys/useWebBuyKeysOrderTotal.ts

```diff
@@ -178,7 +178,7 @@ export function useWebBuyKeysOrderTotal(initialQuantity: number): UseWebBuyKeysO
 
     const getEthButtonText = (): string => {        
         if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
-        if (mintWithEth.isPending || ethMintTx.isPending) {
+        if (mintWithEth.isPending || ethMintTx.isLoading) {
```

Comment:
Why are we using `isLoading` here ?

Reply:
The first object is the write transaction object, the second object is the listen for transaction object.

The two objects are different types and have different interfaces.

Reply:
Thats very confusing :/

Thanks for the info !